03-routing-and-input.Rmd

```diff
@@ -323,4 +324,28 @@ You can print out the names of all of the properties attached to the request by
 
 // TODO: Conflicts between path, query string, body?
 
+### Form submit
+
+One common way of developing API's is to submit a form with its inputs using a POST request. For example to handle a login form we would need: a form to send to the server such as:
+
+```html
+<h2>Basic login form</h2>
+
+<pre>
+    <form action="/login">
```

Comment:
I know the default is POST, but it might be helpful to new folks to specify the `method` tag.

Reply:
Done.